webbpsf/webbpsf_core.py

```diff
@@ -1529,7 +1535,7 @@ def load_wss_opd(self, filename, output_path = None, backout_si_wfe=True, verbos
                 axes[0,1].set_xlabel(f"RMS: {utils.rms(sensing_fp_si_wfe * 1e9, ote_pupil_mask):.2f} nm rms")
 
                 axes[0,2].imshow(opdhdu[0].data + sensing_fp_ote_wfe * ote_pupil_mask , vmin=-vm, vmax=vm, cmap=matplotlib.cm.RdBu_r)
```

Comment:
All the `imshow`  in this block (if plot), should have `origin = 'lower'` so they match the orientation in the example in [https://webbpsf.readthedocs.io/en/latest/jwst_measured_opds.html]